tools/docker-compose.test.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  ao-cu:
+    image: ghcr.io/permaweb/ao-cu:250dbd241864ca2b81e8df7a9ff80e91788b99ac
+    volumes:
+      - ${CU_WALLET_FILE:-./test-wallet.json}:/usr/app/test-wallet.json
+    ports:
+      - ${CU_PORT:-6363}:${CU_PORT:-6363}
+    environment:
+      PORT: ${CU_PORT:-6363}
+      WALLET_FILE: ${CU_WALLET_FILE:-./test-wallet.json}
+      NODE_CONFIG_ENV: ${NODE_CONFIG_ENV:-development}
+      NODE_HEAPDUMP_OPTIONS: ${NODE_HEAPDUMP_OPTIONS:-nosignal}
+      PROCESS_CHECKPOINT_TRUSTED_OWNERS: ${PROCESS_CHECKPOINT_TRUSTED_OWNERS:-fcoN_xJeisVsPXA-trzVAuIiqO3ydLQxM-L4XbrQKzY,WjnS-s03HWsDSdMnyTdzB1eHZB2QheUWP_FVRVYxkXk,-HFe6PleLxj1EdFMYMSetT2NIJioDsZIktn-Y0AwP54}
+      PROCESS_CHECKPOINT_PREFERRED_OWNERS: ${PROCESS_CHECKPOINT_PREFERRED_OWNERS:-HFe6PleLxj1EdFMYMSetT2NIJioDsZIktn-Y0AwP54}
```

Comment:
```suggestion
```

Reply:
not necessary, this is if you only want verified checkpoints from AO (vs. locally)